src/module.mk

```diff
@@ -135,6 +137,60 @@ endif
 LIBFLEXI_SRC := $(sort $(LIBFLEXI_SRC))
 LIBFLEXI_HDR := $(sort $(LIBFLEXI_HDR))
 
+# loop library #########################################################
+LOOP_DIR := $(DIR)/loop_libraries
+
+LOOP_SRC := \
+		$(LOOP_DIR)/library_collier.cpp \
+		$(LOOP_DIR)/loop_library.cpp
+
+LOOP_HDR := \
+		$(LOOP_DIR)/loop_library.hpp \
+		$(LOOP_DIR)/loop_library_interface.hpp
+
+LIBFLEXI_SRC += $(LOOP_SRC)
+LIBFLEXI_HDR += $(LOOP_HDR)
+
+ifeq ($(ENABLE_LOOPTOOLS),yes)
+LIBFLEXI_SRC += \
+		$(LOOP_DIR)/library_looptools.cpp
+LIBFLEXI_HDR += \
+		$(LOOP_DIR)/library_looptools.hpp
+endif
+
+ifeq ($(ENABLE_FFLITE),yes)
+LIBFLEXI_SRC += \
+		$(LOOP_DIR)/library_fflite.cpp
+LIBFLEXI_HDR += \
+		$(LOOP_DIR)/library_fflite.hpp
+endif
+
+ifeq ($(ENABLE_COLLIER),yes)
+LIBFLEXI_SRC += \
+		$(DIR)/loop_libraries/library_collier.cpp
+LIBFLEXI_HDR += \
+		$(DIR)/loop_libraries/library_collier.hpp
+
+$(LOOP_HDR) $(LOOP_SRC) : $(LOOP_DIR)/libcollier_wrapper.a
+
+$(LOOP_DIR)/libcollier_wrapper.a : $(LOOP_DIR)/collier_wrapper.o
+	@echo Building collier wrapper library
+	@ar cr $(LOOP_DIR)/libcollier_wrapper.a $(LOOP_DIR)/collier_wrapper.o
+
+$(LOOP_DIR)/collier_wrapper.mod $(LOOP_DIR)/collier_wrapper.o : $(LOOP_DIR)/collier_wrapper.f90
+ifeq ($(FC),gfortran)
```

Comment:
Please do not hard-code the compiler name here. The user may have set `FC=gfortran-5` or may have given the compiler a totally different name. (Same for `ifort` below.) Please compose such compilation commands in the `configure` script.

Reply:
Inside `configure` default name for `FC` is `gfortran` (line 182).

Inside `configure check_fflags` there is the same check but with additional `*` pattern, like `gfortran*`. **Should I make it in the same way?** 

P.S. Actually, At line 1917 - there is no `ifort*` but only `ifort` - I will fix it later